Lib/email/_parseaddr.py

```diff
@@ -95,6 +95,8 @@ def _parsedate_tz(data):
         return None
     data = data[:5]
     [dd, mm, yy, tm, tz] = data
+    if not (dd and mm and yy):
```

Comment:
nit?
```suggestion
    if not all((dd, mm, yy)):
```

Reply:
It looks less clear to me (and it is also slower). Do you write `sum((dd, mm, yy))` instead of `dd + mm + yy`?